src/ResourceManager/AzureBatch/Commands.Batch/ChangeLog.md

```diff
@@ -20,6 +20,11 @@
 ## Current Release
 * Minor changes for upcoming AzureRM to Az transition
 
+## Version 5.0.0
```

Comment:
@cormacpayne  - Is this all formatted right?

Reply:
@matthchr we haven't finalized our plan for change logs, but I believe we're going to wipe the contents of them all and follow the same format as before. You should be able to get away with keeping everything under the `## Current Release` header since that will probably remain the same (we might change the name of it though)

Reply:
Okay, I moved the stuff that was labeled 5.0.0 to `Current release` for now. Note that this should be 5.0.0 after release though as there are breaking changes.